src/ShopArticleReviewBundle/objects/WebModules/MTPkgShopArticleReviewCore/MTPkgShopArticleReviewCore.class.php

```diff
@@ -100,9 +102,29 @@ protected function GetPkgCommentModuleConfiguration()
             $oPkgCommentModuleConfig->fieldNumberOfCommentsPerPage = 0;
             $oPkgCommentModuleConfig->fieldGuestCommentAllowed = !$oModuleConfiguration->fieldAllowWriteReviewLoggedinUsersOnly;
             $oPkgCommentModuleConfig->fieldPkgCommentTypeId = $this->GetCommentTypeId();
+
+            /**
+             * @psalm-suppress InvalidPropertyAssignmentValue
+             * @FIXME field is `bool` but assigned `int` - This could yield unwanted behaviour, especially with strict checks
```

Comment:
`fieldUseSimpleReporting` and `fieldShowReportedComments` are defined as bool, but are filled with `1` and `0` here - that could potentially lead to errors down the line.